randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -857,7 +876,12 @@ Extra - room_id: [26]
           # TODO: varia / damage reqs
           Morph Ball
           Any of the following:
-              Can Freeze Enemies and Can Use Springball
+              All of the following:
+                  Can Freeze Enemies
+                  Any of the following:
+                      Can Use Springball
+                      # https://youtu.be/3OA51_TEXvc
```

Comment:
```suggestion
                      # Bombjump: https://youtu.be/3OA51_TEXvc
```

Any specific reason this is intermediate?

Reply:
falling down can be pretty punishing since you'll have to start all the way from the beginning, when doing this I thought to myself "I wouldn't want to be required to do this in a race" lol

Reply:
I guess? But sure.